cmd/activator/main.go

```diff
@@ -97,10 +97,6 @@ func statReporter(statSink *websocket.ManagedConnection, stopCh <-chan struct{},
 	for {
 		select {
 		case sm := <-statChan:
-			if statSink == nil {
-				logger.Error("Stat sink is not connected")
-				continue
-			}
 			if err := statSink.Send(sm); err != nil {
 				logger.Errorw("Error while sending stat", zap.Error(err))
 			}
```

Comment:
Should we inline the rest of this code below where it's called? Could make this more apparent?

Reply:
No, I think `main` is already convoluted as is.